src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/OwaspDependencyScanScore.java

```diff
@@ -0,0 +1,135 @@
+package com.sap.sgs.phosphor.fosstars.model.score.oss;
+
+import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.OWASP_DEPENDENCY_CHECK_FAIL_CVSS_THRESHOLD;
+import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.OWASP_DEPENDENCY_CHECK_SCAN_AVAILABILITY;
+import static com.sap.sgs.phosphor.fosstars.model.value.Status.MANDATORY;
+import static com.sap.sgs.phosphor.fosstars.model.value.Status.NOTFOUND;
+import static com.sap.sgs.phosphor.fosstars.model.value.Status.OPTIONAL;
+
+import com.sap.sgs.phosphor.fosstars.model.Score;
+import com.sap.sgs.phosphor.fosstars.model.Value;
+import com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures;
+import com.sap.sgs.phosphor.fosstars.model.qa.ScoreVerification;
+import com.sap.sgs.phosphor.fosstars.model.qa.TestVectors;
+import com.sap.sgs.phosphor.fosstars.model.score.FeatureBasedScore;
+import com.sap.sgs.phosphor.fosstars.model.value.ScoreValue;
+import com.sap.sgs.phosphor.fosstars.model.value.Status;
+import java.io.IOException;
+import java.io.InputStream;
+import org.apache.commons.lang3.Range;
+
+/**
+ * The scores assesses how well an open-source project uses OWASP dependency check to scan
+ * dependencies for known vulnerabilities. It is based on the following features:
+ * <ul>
+ *   <li>{@link OssFeatures#OWASP_DEPENDENCY_CHECK_SCAN_AVAILABILITY}</li>
+ *   <li>{@link OssFeatures#OWASP_DEPENDENCY_CHECK_FAIL_CVSS_THRESHOLD}</li>
+ * </ul>
+ */
+public class OwaspDependencyScanScore extends FeatureBasedScore {
+  
+  /**
+   * The maximum CVSS score.
+   * 
+   * @see <a href="https://nvd.nist.gov/vuln-metrics/cvss">CVSS</a> 
+   */
+  private static final double MAX_CVSS_SCORE = 10.0;
+  
+  /**
+   * The minimum CVSS score.
+   * 
+   * @see <a href="https://nvd.nist.gov/vuln-metrics/cvss">CVSS</a> 
+   */
+  private static final double MIN_CVSS_SCORE = 0.0;
```

Comment:
Please use the existing constants

https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/sgs/phosphor/fosstars/model/value/CVSS.java